errors/daemon.go

```diff
@@ -715,6 +715,15 @@ var (
 		HTTPStatusCode: http.StatusConflict,
 	})
 
+	// ErrorCodeContainerRestarting is generated when we try to top the
+	// container, but the container is restarting.
+	ErrorCodeContainerRestarting = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "CONTAINERRESTARTING",
+		Message:        "Container %s is restarting, wait until the container is running",
+		Description:    "An attempt was made to top the container, but the container is restarting",
```

Comment:
Should we explicitly mention `top` in the description? This error probably apply to other commands as well